ring_buffer/ring_buffer.py

```diff
@@ -5,7 +5,44 @@ def __init__(self, capacity):
     self.storage = [None]*capacity
 
   def append(self, item):
-    pass
+    if self.capacity > self.current:
+      del self.storage[self.current]
+      self.storage.insert(self.current, item)
+      self.current += 1
+
+    elif self.capacity == self.current:
+      self.current = 0
+      self.append(item)
 
   def get(self):
-    pass
\ No newline at end of file
+    array = []
+    for x in self.storage:
+      if x != None:
+        array.append(x)
+    return array
```

Comment:
you could do this in one line as well if you wanted:

return [x for x in self.storage if x is not None]